### ISSUE TITLE
Error in "Beginning Macaulay2"

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2#3309]
I'm noticing recurring github build failures in unrelated branches all stemming from this example:
```m2
i9 : k = toField (QQ[i]/(i^2+1))

o9 = k

o9 : PolynomialRing

i10 : 1/i
stdio:10:2:(3): error: either element not invertible, or no method available to compute its inverse
```
Any ideas what might be going wrong?

**Comment by d-torrance** [comment]
It's something to do with FLINT 3 on Ubuntu 22.04.  It's showing up on different builds now that I uploaded a FLINT 3 package to the PPA...

**Comment by d-torrance** [comment]
I think I narrowed it down to a call to factory -- we ask it to compute the gcd of i and i^2 + 1 and it's returning 0.  Maybe this has something to do with linking against an older version of factory that was linked against FLINT 2.8.4 but having Macaulay2 link against FLINT 3.1.3?

I'll delete the FLINT 3 package from the PPA for Ubuntu 22.04 and just wait for #3288 to be merged so we have 24.04.

**Comment by mahrud** [comment]
I think you also need to update factory, at least that might be a start (Normaliz also links with factory).

**Comment by d-torrance** [comment]
Everything's working again now that the FLINT 3 package is no longer in the PPA.  I figured removing it was easier than backporting factory, etc., especially since it will all be moot once #3288 is merged and we're running everything in Ubuntu 24.0  where FLINT 3 is already available.
